.flake8

```diff
@@ -1,7 +1,7 @@
 [flake8]
 # Rule definitions: http://flake8.pycqa.org/en/latest/user/error-codes.html
 # W503: line break before binary operator
-exclude = venv*,__pycache__,cache
+exclude = venv*,env*,__pycache__,cache
```

Comment:
During `run_tests.sh` `flake8` was picking up .py files from various site-packages unrelated to the project.